pallets/configuration/src/lib.rs

```diff
@@ -577,5 +595,18 @@ pub mod pallet {
         fn target_container_chain_fullness(session_index: T::SessionIndex) -> Perbill {
             Self::config_at_session(session_index).target_container_chain_fullness
         }
+
+        fn max_parachain_cores_percentage(session_index: T::SessionIndex) -> Option<Perbill> {
+            let (past_and_present, _) = Pallet::<T>::pending_configs()
+                .into_iter()
+                .partition::<Vec<_>, _>(|&(apply_at_session, _)| apply_at_session <= session_index);
+
+            let config = if let Some(last) = past_and_present.last() {
+                last.1.clone()
+            } else {
+                Pallet::<T>::config()
+            };
+            config.max_parachain_cores_percentage
```

Comment:
I recently added ` Self::config_at_session(session_index)` as a helper function, could you use it here? :)

Reply:
*fixed*